.bazelrc

```diff
@@ -0,0 +1 @@
+build --test_output=errors
```

Comment:
While I'm not opposed to this change, I would prefer for a final newline to be added so that the file does not end in the middle of a line, which results in weird output when, e.g., `cat`ing the file in a terminal.